data/tutorial/architecture.md

```diff
@@ -29,6 +37,28 @@ the same branch is an atomic and concurrent operation.
 
 Branches are stored in the reference store as pairs of `(hash(commit), branch)`.
 
+As for the block store, you can choose a reference store from the ones provided
+by Irmin, or you can create your own using the interface
+[Irmin.ATOMIC_WRITE_STORE]. The operations you have to implement need to
+satifisy some constraints:
+
+- concurrent reads of the same branch name return the same hash, that is the
+  same commit. Updates of a branch name to a given commit should be consistent:
```

Comment:
'consistent' seems like an odd word here: eventual consistency is not sufficient. Perhaps 'sequentially consistent' instead?